corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -253,6 +253,16 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
                 template: { name: 'widget-multimedia-ko-template', data: $data }"
       >
       </div>
+      <div class="widget-container controls" data-bind="css: controlWidth">
+      <div class="text-danger error-message" data-bind="
+                visible: error,
+                text: error
```

Comment:
I don't see anywhere that would set this attribute for labels.

Reply:
This is probably mimicking the same markup that's used for non-label questions, around line 230. I agree this shouldn't be here if it isn't used, but I'd add a comment noting that labels only deal with server errors, since there's no data type validation for the browser to do.